docs/2-resource-customization.md

```diff
@@ -303,7 +303,7 @@ ActiveAdmin.register Post do
   scope_to :current_user, association_method: :blog_posts
 
   # Finally, you can pass a block to be called:
-  scope_to do
+  scope_to association_method: :itself do
```

Comment:
@faberge-eggs , please add more details I never used this syntax, I just can see that 
https://github.com/activeadmin/activeadmin/blob/d01155f73a0e2cc27d712792270357e2b3759d05/lib/active_admin/resource_controller/scoping.rb#L26 it used here , do not understand 
how `:itself` helps